stdlib/_ast.pyi

```diff
@@ -27,126 +29,180 @@ if sys.version_info >= (3, 8):
     value: expr | None
 
 class Delete(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("targets",)
     targets: list[expr]
 
 class Assign(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("targets", "value", "type_comment")
     targets: list[expr]
     value: expr
 
 class AugAssign(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("target", "op", "value")
     target: expr
     op: operator
     value: expr
 
 class AnnAssign(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("target", "annotation", "value", "simple")
     target: expr
     annotation: expr
     value: expr | None
     simple: int
 
 class For(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("target", "iter", "body", "orelse", "type_comment")
     target: expr
     iter: expr
     body: list[stmt]
     orelse: list[stmt]
 
 class AsyncFor(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("target", "iter", "body", "orelse", "type_comment")
     target: expr
     iter: expr
     body: list[stmt]
     orelse: list[stmt]
 
 class While(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("test", "body", "orelse")
     test: expr
     body: list[stmt]
     orelse: list[stmt]
 
 class If(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("test", "body", "orelse")
     test: expr
     body: list[stmt]
     orelse: list[stmt]
 
 class With(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("items", "body", "type_comment")
     items: list[withitem]
     body: list[stmt]
 
 class AsyncWith(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("items", "body", "type_comment")
     items: list[withitem]
     body: list[stmt]
 
 class Raise(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("exc", "cause")
     exc: expr | None
     cause: expr | None
 
 class Try(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("body", "handlers", "orelse", "finalbody")
     body: list[stmt]
     handlers: list[ExceptHandler]
     orelse: list[stmt]
     finalbody: list[stmt]
 
 class Assert(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("test", "msg")
     test: expr
     msg: expr | None
 
 class Import(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("names",)
     names: list[alias]
 
 class ImportFrom(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("module", "names", "level")
     module: _identifier | None
     names: list[alias]
     level: int
 
 class Global(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("names",)
     names: list[_identifier]
 
 class Nonlocal(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("names",)
     names: list[_identifier]
 
 class Expr(stmt):
+    if sys.version_info >= (3, 10):
+        __match_args__ = ("body",)
```

Comment:
```suggestion
        __match_args__ = ("value",)
```

Reply:
Huh, I wonder why stubtest failed on that one.